packages/devtools_app/lib/src/primitives/auto_dispose_mixin.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
+
```

Comment:
you can remove the extra new lines here. One blank line between copy right and imports is standard.